### PR TITLE
Only import speedcopy on Win > 7

### PR DESCRIPTION
**Pull request by charlesbaynham**
* [x] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.
_Tests pass_

* [x] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here. If the CLI API is changed, I have updated [tab completion scripts](https://github.com/iterative/dvc/tree/master/scripts/completion).
_None required_

* [x] ❌ I will check DeepSource, CodeClimate, and other sanity checks below. (We consider them recommendatory and don't expect everything to be addressed. Please fix things that actually improve code or fix bugs.)

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by charlesbaynham** [comment]
Closes #3259

File: dvc/system.py [hunk 1]
```diff
@@ -10,7 +10,11 @@
 
 logger = logging.getLogger(__name__)
 
-if platform.system() == "Windows" and sys.version_info < (3, 8):
+if (
+    platform.system() == "Windows"
+    and sys.version_info < (3, 8)
+    and sys.getwindowsversion() > (6, 2)
```
Comment (skshetry) [inline dvc/system.py:16]: Should this be `>= (6, 2)`?
I see that [Windows8 is 6.2](https://en.wikipedia.org/wiki/Windows_NT#Releases).

Comment (charlesbaynham) [reply dvc/system.py:16]: Yes you're right, it should (although I haven't tested it on Windows 8)

**Comment by shcheklein** [comment]
@jorgeorpinel should have a Win machine to give it a try :)

**Comment by charlesbaynham** [comment]
> 
> 
> I don't have a Windows machine to test, but looks good. :) Did you try out, @charlesbaynham?

I've tried it out on Win 10 and Win 7: both working now, though I don't have proof that Win 10 is using speedcopy.

**Comment by efiop** [comment]
@charlesbaynham Just curious, why do you still use windows 7? :slightly_smiling_face:

**Comment by jorgeorpinel** [comment]
I'm using Windows now indeed, feel free to contact me any time if you want to run some a test script.

**Comment by charlesbaynham** [comment]
> 
> 
> @charlesbaynham Just curious, why do you still use windows 7? 🙂

It's a lab computer! Somewhere around, I think we still have one running XP...

**Pull request merged**